### PR TITLE
Cp calc branch new

### PR DESCRIPTION
**Pull request by mvandekl**
Hi Andrea, hereby my PR. I think I implemented the changes as discussed. Hope all works!
Best,
Merijn

### CODE REVIEW

File: NTupleMaker/src/Synch17GenTree.cc [hunk 1]
```diff
@@ -57,19 +57,29 @@ void Synch17GenTree::ReadInit(TTree *tree)
    fCurrent = -1;
    fChain->SetMakeClass(1);
 
-   fChain->SetBranchAddress("Higgs_pt", &Higgs_pt, &b_Higgs_pt);
-   fChain->SetBranchAddress("Higgs_eta", &Higgs_eta, &b_Higgs_eta);
-   fChain->SetBranchAddress("Higgs_phi", &Higgs_phi, &b_Higgs_phi);
-   fChain->SetBranchAddress("Higgs_mass", &Higgs_mass, &b_Higgs_mass);
+   fChain->SetBranchAddress("gen_Higgs_pt", &Higgs_pt, &b_Higgs_pt);
```
Comment (cardinia) [inline NTupleMaker/src/Synch17GenTree.cc]: Just for consistency, I would suggest to remove the "gen_" prefix in these variables

**Comment by cardinia** [comment]
Hi Merijn, I added a comment in the Gen Tree definition file: https://github.com/DesyTau/DesyTauAnalysesRun2_25ns/pull/150/files#diff-4ab9b062fcbac333bd3c01f5104362ffR60 

Aside from that I think everything is fine I will just have to find out what the conflicts are

**Comment by mvandekl** [comment]
Hi Andrea, I just removed a few unnecessary GEN prefixed from the Synch17GenTree.cc, and committed and pushed again!
Best,
Merijn

**Comment by cardinia** [comment]
Conflicts solved, merging commits into the HiggsCP branch.

**Pull request merged**